src/main/java/com/github/wovnio/wovnjava/Version.java

```diff
@@ -0,0 +1,23 @@
+package com.github.wovnio.wovnjava;
+
+import java.io.IOException;
+import java.util.Properties;
+
+class Version {
+    private static final String fallbackVersion = "projectPropertiesErrorFallback";
```

Comment:
I would set version to `"unknown"` instead.

Reply:
I'm ok with that